packages/commerce/src/schemas/customer.ts

```diff
@@ -5,14 +5,16 @@ export const getCustomerAddressBodySchema = z.object({
 })
 
 export const customerSchema = z.object({
-  id: z.string(),
-  firstName: z.string(),
-  lastName: z.string(),
-  email: z.string().optional(),
-  phone: z.string().optional(),
-  company: z.string().optional(),
-  notes: z.string().optional(),
-  acceptsMarketing: z.boolean().optional(),
+  customer: z.object({
```

Comment:
Why is it needed to add a nested `customer` object in the customer schema?

Reply:
This I took from another PR that reported that error, but yes, will modify it in the endpoint handler and will make there z.object({customer: customerSchema}), because the operation, returns that as customer obj.